riverrunner/context.py

```diff
@@ -0,0 +1,151 @@
+from riverrunner import settings
+from sqlalchemy import create_engine
+from sqlalchemy.engine.url import URL
+from sqlalchemy import Column, Integer, String, Float, DateTime, ForeignKey
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy.exc import OperationalError
+from sqlalchemy.ext.declarative import declarative_base
+
+
+Base = declarative_base()
+
+
+class Context:
+    """ a connection to the database
+
+    """
+    def __init__(self, connection_string=settings.DATABASE):
+        self.__engine = create_engine(URL(**connection_string))
+
+        self.Session  = sessionmaker()
+        self.Session.configure(bind=self.__engine)
+
+        try:
+            s = self.Session()
+        except OperationalError:
+            print("Unable to connect to destination db")
+            exit(101)
+
+        Base.metadata.create_all(self.__engine)
+
+
+class Address(Base):
+    __tablename__ = 'address'
+
+    latitude  = Column(Float, primary_key=True)
+    longitude = Column(Float, primary_key=True)
+
+    address = Column(String(255))
+    city    = Column(String(255))
+    county  = Column(String(255))
+    state   = Column(String(2))
+    zip = Column(String(10))
+
+    def __repr__(self):
+        return '<Address(latitude="%s", longitude="%s")>' % (self.latitude, self.longitude)
+
+    def __str__(self):
+        return '%s, %s, %s' % (self.address, self.city, self.state)
+
+
+class Measurement(Base):
+    __tablename__ = 'measurement'
+
+    station_id = Column(ForeignKey('station.station_id'), primary_key=True)
+    date_time  = Column(DateTime, primary_key=True)
```

Comment:
just an FYI i changed the type of date_time in the measurement table from `timestamp without time zone` to `timestamp with time zone`, not sure if this affects this line

the changes were because i was running into DST issues when inserting USGS data